lms/templates/emails/activation_email.txt

```diff
@@ -12,29 +14,30 @@ ${_("To get started, please activate your account by clicking on the link below"
   http://${ site }/activate/${ key }
 % endif
 
-% if settings.PLATFORM_NAME == "edX":
-${_("Activation ensures that you can register for {platform_name} courses and"
-      " access the courseware."
-      " If you require assistance, please use our web form at"
-      " {contact_us} or email {info_address}.").format(
-             platform_name=settings.PLATFORM_NAME,
-             contact_us='https://www.edx.org/contact-us',
-             info_address=settings.CONTACT_EMAIL
-      )}
+% if settings.FEATURES.get('IS_EDX_DOMAIN'):
```

Comment:
This is a slight refactor: it's more consistent with the rest of the system to use this feature flag instead of hard-coding a check for the platform name.